gluten-core/src/main/scala/io/glutenproject/extension/ColumnarOverrides.scala

```diff
@@ -560,26 +566,96 @@ case class ColumnarOverrideRules(session: SparkSession)
       overridden
     }
 
+  def fallbackWholeStage(plan: SparkPlan): Boolean = {
+    if (wholeStageFallbackThreshold == -1) {
```

Comment:
How about returning false for every negative threshold?

Reply:
Fixed. Thanks!